cluster/gce/util.sh

```diff
@@ -589,6 +589,22 @@ function kube-up {
   find-release-tars
   upload-server-tars
 
+  if [[ ${KUBE_SHARE_MASTER:-} == "true" ]]; then
```

Comment:
I have no objection to #19218 merging first (happy to rebase); we both see the need to modularize this code.  I made the minimally invasive change, but #19218 is much better.

But if #19218 isn't going to merge soon...